src/front/styles/parents.css

```diff
@@ -0,0 +1,26 @@
+#profile {
+  width: 180px;
+  height: 160px;
+  border-radius: 50%;
+}
+
+#bio {
+  margin-top: 20px;
+  margin-left: 240px;
+  width: 480px;
+  height: 200px;
+}
+
+#form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-size: medium;
+}
+
+#children {
+  margin-right: 20%;
```

Comment:
I feel like this style *might* give you trouble when putting the label and the input side-by-side. I think adding padding is usually better, but that's just a suggestion 🤓

Reply:
i'll ask the rest of group for a solution for this issue